### PR TITLE
spotify: update to 1.1.55

### PR DESCRIPTION
**Pull request by liambabe**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by liambabe** [comment]
The one build request failed because the spotify repository is semi unreliable to download from, It usually takes a couple of attempts.

File: srcpkgs/spotify/template [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Template file for 'spotify'
 pkgname=spotify
-version=1.1.42
+version=1.1.55
 revision=3
```
Comment (abenson) [inline srcpkgs/spotify/template:4]: revision should be reset back to 1.

**Review by ericonr — COMMENTED** [review]
Please remove

```
	_linklib libssl 1.0.0
	_linklib libcrypto 1.0.0
```

and the `_linklib` function itself.

File: srcpkgs/spotify/template (hunk 1 above)
Comment (liambabe) [reply srcpkgs/spotify/template]: Fixed!

**Comment by liambabe** [comment]
> Please remove
> 
> ```
> 	_linklib libssl 1.0.0
> 	_linklib libcrypto 1.0.0
> ```
> 
> and the `_linklib` function itself.

Done!

**Pull request merged**